runtime/dart_isolate.cc

```diff
@@ -1192,6 +1219,8 @@ Dart_Isolate DartIsolate::CreateDartIsolateGroup(
     // NOLINTEND(clang-analyzer-cplusplus.NewDeleteLeaks)
 
     success = InitializeIsolate(embedder_isolate, isolate, error);
+
+    InitDartFFIForIsolateGroup();
```

Comment:
I think this should go above the call to `InitializeIsolate`: The `InitiaializeIsolate` function triggers code that calls into dart code (which theoretically can call `@Native` functions)